processing/src/main/java/io/druid/query/aggregation/hyperloglog/HyperLogLogCollector.java

```diff
@@ -92,8 +92,23 @@ public static HyperLogLogCollector makeLatestCollector()
 
   public static HyperLogLogCollector makeCollector(ByteBuffer buffer)
   {
+    return makeCollector(buffer, false);
+  }
+
+  /**
+   * Create a wrapper object around an HLL sketch contained within a buffer.
+   *
+   * @param buffer      buffer containing HLL sketch
+   * @param bufferOwned if true, we are permitted to change the position and limit of the buffer
```

Comment:
Readability / maintainability. From a program flow standpoint I don't think it makes a difference, but from any new reviewer seeing `HyperLogLogCollector.of(buffer1.duplicate())` and `HyperLogLogCollector.of(buffer2)` with the javadoc signature clarifying that the buffer can have its limit and position modified IMHO is easier to grok than `HyperLogLogCollector.makeCollector(buffer1, false)` vs `HyperLogLogCollector.makeCollector(buffer2, true)`